documentations/mainpage.md

```diff
@@ -126,22 +126,22 @@ User may use command line to unmount file system like this:
 If mounting is done with \ref DOKAN_OPTION_NETWORK, you need Dokan Network Provider for it to work correctly.
 This file (dokannp1.dll) *must* be copied to `%WINDIR%\system32` and you can register the provider on your system with `dokanctl.exe /i n` command.
 
-Without this [Network Provider](https://msdn.microsoft.com/en-us/library/windows/desktop/aa378776%28v=vs.85%29.aspx), Windows Explorer will not handle well Virtual drives mounted as network share and your drive could appears disconnected.
+Without this [Network Provider](https://msdn.microsoft.com/en-us/library/windows/desktop/aa378776%28v=vs.85%29.aspx), Virtual drives mounted as network share will not be handled well by Internet Explorer and your drive could appear disconnected.
 
-If Network Redirector is setup by setting up an UNC Name, Dokan Network Provider will assign UNC Name to the drive label automatically.
+If Network Redirector is setup by setting up an UNC Name, Dokan Network Provider will assign an UNC Name to the drive label automatically.
```

Comment:
I think "a UNC" is correct. It's the pronunciation that determines a/an.

Reply:
That is correct. Although `U` is a vowel, `a UNC` is correct in this context. A related article is [here](https://www.ecenglish.com/learnenglish/when-use-a)